pages/data-migration/best-practices.mdx

```diff
@@ -66,20 +66,26 @@ href="/fundamentals/indexes"
 
 ### Use in-memory analytical storage mode
 
-Memgraph supports three storage modes: in-memory transactional, in-memory
-analytical and on-disk transactional. **To speed up import, use in-memory
-analytical storage mode**. On a larger scale, if you stick with the default
-in-memory transactional storage mode, you’ll experience slower import time and
-spend more resources during data import.  
+Memgraph supports three storage modes: `IN_MEMORY_TRANSACTIONAL`, `IN_MEMORY_ANALYTICAL` and `ON_DISK_TRANSACTIONAL`
+**To speed up import, use in-memory analytical storage mode**. 
```

Comment:
@antejavor Is there any specific reason why you decided for `IN_MEMORY_ANALYTICAL` (and similar) above and left in-memory analytical storage below? Do you think it was wrong to say in-memory transactional, in-memory
analytical and on-disk transactional? I like it when it's consistent.

Reply:
I do not think it was wrong to say, I just prefer this notation since it is pretty important and to me a bit more visible/important in the code format. Changing a storage mode is a big deal.

Reply:
I still think we should stick to one notation and link to the instructions on how to change the storage mode if necessary (it's already linked at the bottom of the section). If you want to emphasize it, feel free to put it in bold.

Reply:
I reviewed our current docs a bit, and you're right, `IN_MEMORY_TRANSACTIONAL`, `IN_MEMORY_ANALYTICAL` and `ON_DISK_TRANSACTIONAL` was often used. So, if you prefer that one, let's stick with that :)